packages/cli/src/commands/build.js

```diff
@@ -78,6 +79,7 @@ export default {
     // TODO: remove if in Nuxt 3
     if (nuxt.options.mode === MODES.spa && nuxt.options.target === TARGETS.server && cmd.argv.generate !== false) {
       // Build + Generate for static deployment
+      consola.info(`Enforcing ${chalk.bold.blue('static')} target because of \`nuxt build\` with server-side rendering disabled`)
```

Comment:
Maybe also add a warning to use `nuxt generate` and this will be unsupported?

Reply:
Good idea